Frameworks/GLKit/GLKEffect.mm

```diff
@@ -528,7 +529,6 @@ - (BOOL)prepareShaders {
         self.shader = [[GLKShaderCache get] addShaderNamed:self.shaderName source:p];
         if (self.shader == nil) {
             NSTraceError(TAG, @"There was a problem generating a shader for material %@", self.shaderName);
-            [p release];
```

Comment:
```addShaderNamed:source:``` doesn't retain ```p``` so this is missing a release in the non-error case were ARC to be left disabled